### ISSUE TITLE
Add analysis for Java classes that are not :import-ed 

### ISSUE THREAD

**Issue by borkdude** [clj-kondo/clj-kondo#1633]
E.g.:

```
(ns foo)
clojure.lang.PersistentVector
```

```
$ clojure -M:clj-kondo/dev --lint - <<< 'foo.bar.Baz' --config '{:output {:analysis {:java-class-usages true} :format :edn}}'
{:findings [], :summary {:error 0, :warning 0, :info 0, :type :summary, :duration 34, :files 1}, :analysis {:namespace-definitions [], :namespace-usages [], :var-definitions [], :var-usages [{:end-row 1, :name-end-col 12, :name-end-row 1, :name-row 1, :name foo.bar.Baz, :filename "<stdin>", :from user, :col 1, :name-col 1, :end-col 12, :row 1, :to :clj-kondo/unknown-namespace}], :java-class-usages []}}
``` 

The `foo.bar.Baz` usage is incorrectly analyzed as a var usage and should be moved to java-class-usages.

**Comment by pedrorgirardi** [comment]
Hi there! 👋 

I'm testing the new Java definitions & usages analysis and I noticed that `java.util.Date` appears in `var-usages`:

<img width="238" alt="Screen Shot 2022-03-28 at 07 56 41" src="https://user-images.githubusercontent.com/585191/160385579-b02bbd87-3126-4908-af46-f8d47351df51.png">

<img width="740" alt="Screen Shot 2022-03-28 at 07 54 00" src="https://user-images.githubusercontent.com/585191/160385596-5d49791d-0d18-4663-a133-49dfe0b66df8.png">

**Comment by borkdude** [comment]
@pedrorgirardi Should be fixed now, please test

**Comment by pedrorgirardi** [comment]
Thanks! Will do.

**Comment by pedrorgirardi** [comment]
Seems fine:

```clojure
(import (java.util Date))

Date

java.util.Date
```

**Analysis:**

```python
{'java-class-usages': [{'class': 'java.util.Date',
                        'col': 20,
                        'end-col': 24,
                        'end-row': 1,
                        'filename': '/Users/pedro/Library/Application '
                                    'Support/Sublime '
                                    'Text/Packages/Pep/tests/java_interop.clj',
                        'row': 1,
                        'uri': None},
                       {'class': 'java.util.Date',
                        'col': 1,
                        'end-col': 5,
                        'end-row': 3,
                        'filename': '/Users/pedro/Library/Application '
                                    'Support/Sublime '
                                    'Text/Packages/Pep/tests/java_interop.clj',
                        'name-col': 1,
                        'name-end-col': 5,
                        'name-end-row': 3,
                        'name-row': 3,
                        'row': 3,
                        'uri': None},
                       {'class': 'java.util.Date',
                        'col': 1,
                        'end-col': 15,
                        'end-row': 5,
                        'filename': '/Users/pedro/Library/Application '
                                    'Support/Sublime '
                                    'Text/Packages/Pep/tests/java_interop.clj',
                        'row': 5,
                        'uri': None}],
 'keywords': [],
 'local-usages': [],
 'locals': [],
 'namespace-definitions': [],
 'namespace-usages': [],
 'var-definitions': [],
 'var-usages': [{'arity': 1,
                 'col': 1,
                 'end-col': 26,
                 'end-row': 1,
                 'filename': '/Users/pedro/Library/Application Support/Sublime '
                             'Text/Packages/Pep/tests/java_interop.clj',
                 'from': 'user',
                 'macro': True,
                 'name': 'import',
                 'name-col': 2,
                 'name-end-col': 8,
                 'name-end-row': 1,
                 'name-row': 1,
                 'row': 1,
                 'to': 'clojure.core',
                 'varargs-min-arity': 0}]}

```

**Comment by borkdude** [comment]
The `:uri` looks weird in your case...

**Comment by borkdude** [comment]
If you can reconstruct a call in which this uri is nil, it might be fixable.

**Comment by pedrorgirardi** [comment]
Isn't it `nil` already? It's being converted from JSON to Python data structures.

**Comment by borkdude** [comment]
Yes, but I mean, the call which results into this nil uri. A repro.

**Comment by pedrorgirardi** [comment]
Ah, gotcha! Let me work a bit on the plugin and I will have a better report to give you :)
